.test-home/.gitignore

```diff
@@ -1 +1,5 @@
-.warpforge/*
+.warpforge/warehouse/*
+.warpforge/cache
+.warpforge/catalogs
+.warpforge/runc-root
+!.warpforge/warehouse/4z9/
```

Comment:
Don't really need, if it's a single asset in here and you added it (or `add -f`'d it) already.

Might be worth a comment either here or in a spare file in that directory _why_ something's there that's gitignored, though, if we're worried about the clarity of it.

Reply:
This dir should have been removed. Oops.